.readthedocs.yaml

```diff
@@ -0,0 +1,27 @@
+# This file is part of REANA.
+# Copyright (C) 2023 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.8"
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  configuration: docs/conf.py
+
+# It is recommended to specify the dependencies to enable reproducible builds:
```

Comment:
Alternatively, since the file is rather short and mostly self-explanatory anyway, what about removing all the documentation lines? A little bit like we do for the GitHub Actions configurations. I think the file would be readable enough as such:

```yaml
# This file is part of REANA.
# Copyright (C) 2023 CERN.
#
# REANA is free software; you can redistribute it and/or modify it
# under the terms of the MIT License; see LICENSE file for more details.

version: 2

build:
  os: ubuntu-22.04
  tools:
    python: "3.8"

sphinx:
  configuration: docs/conf.py

python:
  install:
    - requirements: docs/requirements.txt
```